### PR TITLE
Azarembok/veda heal

### PR DESCRIPTION
**Pull request by azarembok**
This PR enables the veda heal tool (including very simple UI accessible from the /heal URL) and support within the delivery worker.  Changes here include those made by other developers that had not been merged, plus some minor fixes from me.

### CODE REVIEW

**Comment by azarembok** [comment]
@fysheets please hold off on reviewing for now, I will need some time to work out the build issues.

**Comment by azarembok** [comment]
@fysheets the build/test issues look sorted out, you can review this when you get a chance.  I'll squash my commits before merging.

File: control/veda_heal.py [hunk 1]
```diff
@@ -106,18 +105,20 @@ def send_encodes(self):
                 veda_id = v.edx_id
                 encode_profile = encode
                 job_id = uuid.uuid1().hex[0:10]
-                task_result = celeryapp.worker_task_fire.apply_async(
-                    (veda_id, encode_profile, job_id),
-                    queue=self.auth_dict['celery_worker_medium_queue'].strip(),
-                    connect_timeout=3
-                )
+                # TODO: Commenting this out temporarily.  The worker_task_fire task
```
Comment (fysheets) [inline control/veda_heal.py]: For the official record, why did this need to be commented out / what was breaking with it included?

File: templates/heal.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```
Comment (fysheets) [inline templates/heal.html]: This shouldn't be too hard to make prettier :)

File: control/veda_heal.py (hunk 1 above)
Comment (azarembok) [reply control/veda_heal.py]: We had a circular dependency between veda_heal.py and celeryapp.py (both were importing each other), and this was causing errors when a 3rd module tried to import veda_heal.  To remove the dependency on celeryapp I removed the call above.

**Pull request merged**